docs/how-tos/production_configuration.rst

```diff
@@ -0,0 +1,245 @@
+
+.. note::
+
+    Don't forget the usual checklist items! Make sure the server is secured and backed up! Make sure you understand the basics of `superset security configuration`_ and have updated your settings appropriately if necessary. Aspects does a lot with user roles and permissions to support localized dashboards, if you need help understanding how it all fits together please reach on in #aspects on the Open edX Slack!
+
+
+Monitoring
+##########
+
+There are a few key metrics worth monitoring to make sure that Aspects is healthy:
+
+**ClickHouse Lag Time**
+
+This is the time between now and the last xAPI event arriving. The frequency of events depends on a lot of factors, but an unusually long lag can mean that events aren't arriving. An easy way to check this is by querying ClickHouse with a query such as
+
+.. code-block:: sql
+
+    SELECT
+        count(*) as ttl_count,
+        max(emission_time) as most_recent,
+        date_diff('second', max(emission_time), now()) as lag_seconds
+    FROM xapi.xapi_events_all
+    FINAL
+    FORMAT JSON
+
+
+**Celery Queue Length**
+
+If you are using Celery it's important to make sure that the queue isn't growing uncontrollably due to the influx of new events and other tasks associated with Aspects. For a default install the following Python will show you the the number of tasks waiting to be handled for the LMS and CMS queues:
+
+.. code-block:: python
+
+        r = redis.Redis.from_url(settings.BROKER_URL)
+        lms_queue = r.llen("edx.lms.core.default")
+        cms_queue = r.llen("edx.cms.core.default")
+
+
+**Redis Bus Queue Length**
+
+For redis streams you can find the number of pending items using the following Python:
+
+.. code-block:: python
+
+        r = redis.Redis.from_url(settings.EVENT_BUS_REDIS_CONNECTION_URL)
+
+        # "analytics" is the topic, your configuration may vary
+        info = r.xinfo_stream("analytics", full=True)
+
+        lag = 0
+
+        # You may prefer to break out the lag here by consumer group
+        try:
+            for g in info["groups"]:
+                lag += g["lag"]
+        # Older versions of redis don't have "lag".
+        except KeyError:  # pragma: no cover
+            pass
+
+        return lag
+
+
+**Kafka Bus**
+
+If you are running Kafka you likely have other tools for monitoring and managing the service. Generally you are looking for the difference between the low and high watermark offsets for each partition in your configured topic and consumer group to determine how many messages each partition has processed vs the total.
+
+**Superset**
+
+Superset is a fairly standard Flask web application, and should be monitored for the usual metrics. So far the only slowness we have encountered has been with slow ClickHouse queries.
```

Comment:
Domain sharding isn't supported by aspects, however a pretty good alternative is to use HTTP 2/3 for the Superset Deployments. It will improve the success of concurrent requests in Superset Dashboards.

This can be pretty complex to setup, so there shouldn't be any specific recommendation. Only the warning

Reply:
We've added this by default now, right?

Reply:
Yes, we've resolved all issues around this